src/css/blocks/product-card.css

```diff
@@ -0,0 +1,122 @@
+.card {
+  display: flex;
+  flex-direction: column;
+
+  font-family: Roboto, sans-serif;
```

Comment:
add font-family with `html` selector

Reply:
Now is Done
```
html {
  font-family: "Roboto", sans-serif;
  font-weight: 400;
  color: var(--main-color);
}
```

font-family better use width HTML or BODY teg ???